honeybee_radiance/dictutil.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+"""Utilities to convertint any dictionary to Python objects.
+
+Note that importing this module will import almost all modules within the
+library in order to be able to re-serialize almost any dictionary produced
+from the library.
+"""
+from honeybee_radiance.putil import dict_to_primitive
+from honeybee_radiance.modifierset import ModifierSet
+from honeybee_radiance.sensorgrid import SensorGrid
+from honeybee_radiance.view import View
+from honeybee_radiance.lightsource.dictutil import dict_to_light_source, \
+    LIGHT_SOURCE_TYPES
+
+
+def dict_to_object(honeybee_radiance_dict, raise_exception=True):
+    """Re-serialize a dictionary of almost any object within honeybee_radiance.
+
+    This includes any Modifier, ModifierSet, LightSource, SensorGrid or View object.
+
+    Args:
+        honeybee_radiance_dict: A dictionary of any Honeybee radiance object. Note
+            that this should be a non-abridged dictionary to be valid.
+        raise_exception: Boolean to note whether an excpetion should be raised
+            if the object is not identified as a part of honeybee_radiance.
+            Default: True.
+
+    Returns:
+        A Python object derived from the input honeybee_radiance_dict.
+    """
+    try:  # get the type key from the dictionary
+        obj_type = honeybee_radiance_dict['type']
+    except KeyError:
+        raise ValueError('Honeybee_radiance dictionary lacks required "type" key.')
+
+    if obj_type == 'ModifierSet':
```

Comment:
I think this method can also be simplified similar to the comment that I left for lightsources. At least for the first three objects.

Reply:
The strategy above definitely .makes sense for the lightsource dictutil module but I'm a little less inclined to do this here since it will most likely harm the performance given that it's much more likely that someone is re-serializing a primitive than  any of these 3 object. So the `except` case isn't really exceptional.  If we add many more classes for this module to handle, then I'm ok with changing it. But 3 doesn't seem like enough.